src/main/java/christmas/domain/Order.java

```diff
@@ -0,0 +1,35 @@
+package christmas.domain;
+
+import christmas.utils.validators.OrderValidator;
+import java.util.Collections;
+import java.util.List;
+
+public class Order {
+    private final List<OrderItem> orderItems;
+
+    private Order(final List<OrderItem> orderItem) {
+        this.orderItems = orderItem;
+    }
+
+    public static Order from(final List<OrderItem> orderItems, OrderValidator orderValidator) {
+        orderValidator.validate(orderItems);
+        return new Order(orderItems);
+    }
+
+    public int requestTotalPrice() {
+        return orderItems.stream().mapToInt(item -> item.requestPrice()).sum();
+    }
```

Comment:
이 부분은 취향 차이일 것 같으니 편하게 들어주시면 감사할 것 같아요.
저는 스트림이 반복문과 비교해 가질 수 있는 최대 장점이 가독성이라고 생각해요.
그리고 위와 같은 스트림 체인은 각 단계마다 개행을 시켜 주어 단계 구분을 확실하게 해주면서 더 가독성을 높여주는 편을 선호합니다!

컨벤션에서 역시 가독성을 위한 스트림 체인의 개행을 허용해 주고 있으며, 이 경우 `.` 앞에서 개행을 해줍니다.
```suggestion
    public int requestTotalPrice() {
        return orderItems.stream()
                .mapToInt(OrderItem::requestMainCount)
                .sum();
    }
```